app/dv/pom.xml

```diff
@@ -354,7 +353,7 @@
       <dependency>
         <groupId>org.mongodb</groupId>
         <artifactId>mongo-java-driver</artifactId>
-        <version>${version.org.mongodb}</version>
+        <version>${mongodb.version}</version>
```

Comment:
cc @rareddy

Reply:
More cc of people mentioned in pom, just in case @rareddy is busy: @blafond @shawkins @mdrillin

Reply:
Solved in #7731